Engine/source/gui/controls/guiMLTextCtrl.cpp

```diff
@@ -768,7 +768,7 @@ void GuiMLTextCtrl::onMouseDown(const GuiEvent& event)
    mSelectionAnchorDropped = event.mousePoint;
    if (mSelectionAnchor < 0)
       mSelectionAnchor = 0;
-   else if (mSelectionAnchor >= mTextBuffer.length())
+   else if (mSelectionAnchor >= mTextBuffer.length() - 1)
```

Comment:
Oh ok so it looks like your change should have the same behavior, there was no -1 for mSelectionAnchor assignment. I actually noticed we made several little -1 changes to selection stuff as the selection had odd behavior the specifics of each change was back before we use version control and I think dates back to TGE so can't dig those up, but since this change should have the same result then that works. I will make the change.